### DISCUSSION TITLE
Prosposal: support for break-inside

### DISCUSSION THREAD

**Discussion by tomtomau** [tailwindlabs/tailwindcss#1729] [Ideas]
Hi there, I've found tailwind is super easy to use for styling pages for print with the `print` screen config!

It would be great if there was support for `break-inside` properties: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/break-inside)

I couldn't find any other issues or pull requests referencing this property.

**Comment by JeffBeltran** [comment]
we are a sad bunch who find ourselves here. Would be a nice add at some point though

**Comment by austriker27** [comment]
I would also love to see this! Its a great semi-secret CSS property to help build native masonry grids.
